server_test.go

```diff
@@ -178,7 +186,7 @@ func TestStubServer_RoutesRequest(t *testing.T) {
 			&http.Request{Method: "POST",
 				URL: &url.URL{Path: "/v1/invoices/in_123/pay"}})
 		assert.NotNil(t, route)
-		assert.Equal(t, chargeDeleteMethod, route.operation)
+		assert.Equal(t, invoicePayMethod, route.operation)
```

Comment:
My tweaks to the test fixtures revealed that these were previously wrong, so they've also been fixed.